tests/tests/test_timetable.py

```diff
@@ -160,3 +160,95 @@ def test_get_timetable(
     ]
     actual_results = [_TrainScheduleSummary(**train_schedule) for train_schedule in timetable.train_schedule_summaries]
     recursive_approx([x.__dict__ for x in expected_schedules], [x.__dict__ for x in actual_results])
+
+
+def test_timetable_import(small_scenario: Scenario, fast_rolling_stock: int):
+    _ = fast_rolling_stock  # We use it by name
```

Comment:
What's the point of this assignment? Would "del fast_rolling_stock" be clearer?

Reply:
Maybe, the point was to prevent a potential lint about it being unused